client/layout/masterbar/style.scss

```diff
@@ -38,7 +43,7 @@ $autobar-height: 20px;
 	display: flex;
 		align-items: center;
 	position: relative;
-	color: $white;
+	@include theme( color, --masterbar-color );
```

Comment:
The include does a bit more behind the scenes. In addition to adding the themeable variable, the include outputs a color fallback. As outlined in a bit more detail in `p4TIVU-75d-p2 (internal reference)` we need to provide a color fallback (the default color scheme) for browsers that do not support CSS custom properties yet.

Therefore `@include theme( color, --masterbar-color );` compiles to
```
color: $masterbar-color;
color: var( --masterbar-color  );
```

This way we ensure that there is always a fallback provided for each custom property.